src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs

```diff
@@ -437,4 +437,218 @@ internal enum WebRTCDisplayType
 
         Evas,
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// Specifies the type of WebRTC statistics.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    [Flags]
+    public enum WebRTCStatisticsCategory
+    {
+        /// <summary>
+        /// Codec.
+        /// </summary>
+        Codec = 0x0001,
+
+        /// <summary>
+        /// Inbound RTP.
+        /// </summary>
+        InboundRtp = 0x0002,
+
+        /// <summary>
+        /// Outbound RTP.
+        /// </summary>
+        OutboundRtp = 0x0004,
+
+        /// <summary>
+        /// Remote inbound RTP.
+        /// </summary>
+        RemoteInboundRtp = 0x0008,
+
+        /// <summary>
+        /// Remote Outbound RTP.
+        /// </summary>
+        RemoteOutboundRtp = 0x0010,
+
+        /// <summary>
+        /// All types of WebRTC statistics.
+        /// </summary>
+        All = Codec | InboundRtp | OutboundRtp | RemoteInboundRtp | RemoteOutboundRtp
+    }
+
+    [Flags]
+    internal enum WebRTCStatisticsPropertyCategory
+    {
+        Common = 0x00000100,
+
+        Codec = 0x00000200,
+
+        RtpStream = 0x00000400,
+
+        ReceivedRtpStream = 0x00000800,
+
+        InboundRtpStream = 0x00001000,
+
+        SentRtpStream = 0x00002000,
+
+        OutboundRtpStream = 0x00004000,
+
+        RemoteInboundRtpStream = 0x00008000,
+
+        RemoteOutboundRtpStream = 0x00010000
+    }
+
+    /// <summary>
+    /// Specifies the WebRTC statistics property.
+    /// </summary>
+    /// <since_tizen> 10 </since_tizen>
+    public enum WebRTCStatisticsProperty
+    {
+        /// <summary>
+        /// Timestamp.
+        /// </summary>
+        Timestamp = WebRTCStatisticsPropertyCategory.Common | 0x01,
+
+        /// <summary>
+        /// ID.
+        /// </summary>
+        Id = WebRTCStatisticsPropertyCategory.Common | 0x02,
+
+        /// <summary>
+        /// Payload type.
+        /// </summary>
+        PayloadType = WebRTCStatisticsPropertyCategory.Codec | 0x01,
+
+        /// <summary>
+        /// Clock rate.
+        /// </summary>
+        ClockRate = WebRTCStatisticsPropertyCategory.Codec | 0x02,
+
+        /// <summary>
+        /// Channels.
```

Comment:
a number of channels, a count?

Reply:
Yes it's the number of channels. I'll change the description to `The number of channels`.
Please refer https://www.w3.org/TR/webrtc-stats/#dictionary-rtccodecstats-members for more details.